### PR TITLE
BrenKt-SK-1243-ALL-WEB-Exchange-shared_account

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.fr-fr.md [hunk 1]
```diff
@@ -31,10 +31,20 @@ Renseignez les champs demandés :
...
-|Quota|Indiquez le quota de stockage souhaité pour votre compte partagé. Le quota maximal est de **10 Go pour l'ensemble de vos comptes partagés**.|
+|Quota|Indiquez le quota de stockage souhaité pour votre compte partagé, [selon l'espace disponible](#size).|
 |Nom à afficher|Le nom d'affichage que vous souhaitez faire apparaître lors d'un envoi depuis votre compte partagé.|
 |Cacher l'adresse dans l'annuaire|Cacher l’adresse dans l’annuaire permet de faire en sorte que l’adresse du compte partagé ne soit pas visible dans la liste des adresses de votre plateforme Exchange.|
 
+<a name="#size"></a>
+
+> [!primary]
+>
+> L'espace disponible pour créer un compte partagé dépend du nombre de comptes souscrits à votre plateforme. Chaque compte Exchange souscrit à votre plateforme débloque 5 Go pour les comptes partagés.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.fr-fr.md]:
```suggestion
> L'espace disponible pour créer un compte partagé dépend du nombre de comptes souscrits sur votre plateforme. Chaque compte Exchange souscrit sur votre plateforme débloque 5 Go pour les comptes partagés.
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.es-es.md [hunk 2]
```diff
@@ -35,10 +35,20 @@ Introduzca los campos solicitados:
...
-|Límites|Indique el límite de almacenamiento que desea asignarle a su cuenta compartida. La capacidad máxima es de **10 GB para todas sus cuentas compartidas**.|
+|Cuota|Indique la cuota de almacenamiento deseada para su cuenta compartida, [en función del espacio disponible](#size).|
 |Nombre mostrado|El nombre mostrado que quiere que aparezca en un envío desde su cuenta compartida.|
 |Ocultar la dirección en el directorio|Ocultar la dirección en el directorio permite evitar que la dirección de la cuenta compartida sea visible en la lista de direcciones de su plataforma Exchange.|
 
+<a name="#size"></a>
+
+> [!primary]
+>
+> El espacio disponible para crear una cuenta compartida depende del número de cuentas contratadas a su plataforma. Cada cuenta Exchange suscrita a su plataforma desbloquea 5 GB para las cuentas compartidas.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.es-es.md]:
```suggestion
> El espacio disponible para crear una cuenta compartida depende del número de cuentas contratadas en su plataforma. Cada cuenta Exchange suscrita en su plataforma desbloquea 5 GB para las cuentas compartidas.
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.en-gb.md [hunk 3]
```diff
@@ -31,10 +31,20 @@ Fill in the requested fields:
...
-|Quota|Enter the storage quota you want for your shared account. The maximum quota is **10 GB for all of your shared accounts**.|
+|Quota|Enter the desired storage quota for your shared account, [depending on available space](#size).|
 |Display name|The display name you want to appear when sending from your shared account.|
 |Hide the address in the directory|By hiding the address in the directory you can ensure that the shared account is not visible in the address list of your Exchange service.|
 
+<a name="#size"></a>
+
+> [!primary]
+>
+> The space available for creating a shared account depends on the number of accounts subscribed to your platform. Each Exchange account subscribed to your platform unblocks 5 GB for shared accounts.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.en-gb.md]:
```suggestion
> The space available for creating a shared account depends on the number of accounts subscribed on your platform. Each Exchange account subscribed on your platform unblocks 5 GB for shared accounts.
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.en-gb.md [hunk 4]
```diff
@@ -31,10 +31,20 @@ Fill in the requested fields:
 |Field|Description|
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_shared_account/guide.en-gb.md:31]: Please update the date

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**